distribution/docs/src/main/resources/content/_configuring/configuring-data-policy-plugins.adoc

```diff
@@ -0,0 +1,61 @@
+:title: Configuring Data Policy Plugins
+:type: configuration
+:status: published
+:summary: Configuring policy plugins
+:parent: Configuring Data Management
+:order: 05
+
+== {title}
+
+Configure the data-related policy plugins to determine the accessibility of data held by ${branding}.
+
+=== Configuring the Metacard Attribute Security Policy Plugin
+
+The Metacard Attribute Security Policy Plugin combines existing metacard attributes to make new attributes and adds them to the metacard.
+
+. Navigate to the *${admin-console}*.
+. Select the *${ddf-catalog}* application tile
+. Select the *Configuration* tab
+. Select the *Metacard Attribute Security Policy Plugin*.
+
+Sample configuration of the <<Metacard Attribute Security Policy Plugin>>.
+
+To configure the plugin to combine the attributes `sourceattribute1` and `sourceattribute2` into a new
+attribute `destinationattribute1` using the union,
+enter these two lines under the title **Metacard Union Attributes**
+
+|===
+|Metacard Union Attributes
+
+|`sourceattribute1=destinationattribute1`
+
+|`sourceattribute2=destinationattribute1`
+|===
+
+See <<org.codice.ddf.catalog.security.policy.metacard.MetacardAttributeSecurityPolicyPlugin,Metacard Attribute Security Policy Plugin configurations>> for all possible configurations.
+
+=== Configuring the Metacard Validity Filter Plugin
+
+The <<Metacard Validity Filter Plugin>> determines whether metacards with validation errors or warnings are filtered from query results.
+
+. Navigate to the *${admin-console}*.
+. Select the *${ddf-catalog}* application.
+. Select the *Configuration* tab.
+. Select the *Metacard Validity Filter Plugin*.
+.. Check *Filter Errors* to hide metacards with errors from users.
+.. Check *Filter Warnings* to hide metacards with warnings from users.
+
+See <<ddf.catalog.metacard.validation.MetacardValidityFilterPlugin,Metacard Validity Filter Plugin configurations>> for all possible configurations.
```

Comment:
Consider adding a reference to the MetacardValidityMarkerPlugin. It is the thing that adds the validation errors and warnings to the metacards.

Reply:
will add link